.github/workflows/branch-validation.yml

```
@@ -0,0 +1,98 @@
+name: Branch Validation
+
+on:
+  push:
+    branches:
+      - 'develop'
+      - 'feat/*'
+      - 'fix/*'
+      - 'refactor/*'
+      - 'chore/*'
+      - 'release/*'
+      - 'hotfix/*'
```

I think we should also add `ci/*`

```suggestion
on:
  push:
    branches:
      - 'develop'
      - 'feat/*'
      - 'fix/*'
      - 'refactor/*'
      - 'chore/*'
      - 'release/*'
      - 'hotfix/*'
      - 'ci/*'
```